exploring_UV_extinction_curves_pt1/downloading_UV_data_pt1.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>Similar to the above comment... it's weird because we kind of already "know" which one we want to choose to make the tutorial work correctly. But it can be useful to "pretend" that you don't already know so you can model the procedure to the user. So I'd recommend also showing how to find the correct line, if you didn't already know it was row 1144. (Also that makes sure it's robust if for some reason the ordering of the rows changes in a future update or something!)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>